scripts/artifacthub/hub.go

```diff
@@ -214,6 +218,17 @@ func addArtifactHubMetadata(sourceDirectory, destinationPath, ahBasePath string,
 	}
 }
 
+func checkVersion(artifactHubMetadata *ArtifactHubMetadata, constraintTemplate map[string]interface{}, newConstraintTemplateHash string) {
+	// compare hash
+	if artifactHubMetadata.Digest != newConstraintTemplateHash {
+		// compare version
+		if artifactHubMetadata.Version == fmt.Sprintf("%s", constraintTemplate["metadata"].(map[string]interface{})["annotations"].(map[string]interface{})["metadata.gatekeeper.sh/version"]) {
```

Comment:
this would require version changes for metadata-only additions like https://github.com/open-policy-agent/gatekeeper-library/pull/251

Is that a desired outcome?

Reply:
From the version discussion doc and #258, we currently always require a version bump. 

IMO whenever the template.yaml file is updated, we should require an update in the version, otherwise it's pretty hard to decide when to bump the version and not (it will always require humans/approvers making a call and it might not always be consistent).